ethicalads-theme/templates/ea/homepage.html

```diff
@@ -24,12 +24,13 @@
 
         <!-- Heading -->
         <h1 class="display-3 font-weight-light" data-aos="fade-left">
-          <span class="d-block">Respectful Advertising</span>
-          <span class="d-block" data-aos-delay="100">For Developers</span>
+          <span class="d-block">Transparent Advertising</span>
+          <span class="d-block">For Developers</span>
+          <span class="d-block" data-aos-delay="100">That Works</span>
         </h1>
 
         <!-- Text -->
-        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">EthicalAds is a privacy-preserving ad network targeting developers</p>
+        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">For everyone tired of ads that track you, EthicalAds serves privacy-preserving ads to developers</p>
```

Comment:
This feels pretty publisher-focused and possibly offputting to advertisers. 

```suggestion
        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">We get hand-picked advertisers in front of developers with privacy-preserving ads on our high-quality network</p>
```

Perhaps a bit overly verbose though.